administration/src/util/qrcode.ts

```diff
@@ -246,3 +248,21 @@ export const drawQRCode = (
     size
   )
 }
+
+export const convertProtobufToHexCode = (qrCode: QrCode): string => {
+  const qrCodeMatrix = encodeQRCode(qrCode.toBinary()).getMatrix()
+  return qrCodeMatrix
+    .getArray()
+    .map(row => BigInt(`0b${row.join('')}`).toString(16))
+    .join('-')
+}
+
+export const convertHexmapToUInt8Array = (hexmap: string): Uint8Array[] => {
+  const zeroASCIICode = 48
+  const hexRows = hexmap.split('-')
+  const binaryStringMatrix = hexRows.map(hex => BigInt(`0x${hex}`).toString(2).padStart(hexRows.length, '0'))
+  const binaryUInt8Matrix = binaryStringMatrix.map(stringRow =>
+    Uint8Array.from(stringRow.split(''), x => x.charCodeAt(0) - zeroASCIICode)
```

Comment:
can you explain to me why we need this zeroAsciiCode stuff?
What is that better than just converting the string to an int?
`Uint8Array.from(stringRow.split(''), x => Number(x))`

Reply:
This also works and is a lot nicer. Thanks for the hint, i adjusted it.